### PR TITLE
Multisite tutorial

### PR DESCRIPTION
**Pull request by Jay-umbr**
Essentially - how to have 2+ websites on one project.
Not recommended which I highlight several times

### CODE REVIEW

File: Tutorials/Multisite-Setup/index.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+meta.Title: "Multisite setup in Umbraco"
+product: "CMS"
+meta.Description: "A guide to multisite setup in Umbraco"
+---
+
+# Multisite setup on Umbraco Cloud
+
+This tutorial explains how to host multiple sites from one project/installation of Umbraco.
+For practical reasons, we recommend using [Baselines](../../Umbraco-Cloud/Getting-Started/Baselines/) on the Cloud projects.
+If you are planning to create a multilanguage site in Umbraco 8, please take a look at the [Multilanguage Setup](../Multilanguage-Setup/) tutorial instead.
```
Comment (sofietoft) [inline Tutorials/Multisite-Setup/index.md:12]:
```suggestion

If you are planning to create a multilingual site in Umbraco 8, please take a look at the [Multilanguage Setup](../Multilanguage-Setup/) tutorial instead.
```

File: Tutorials/Multisite-Setup/index.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+versionFrom: 8.0.0
+meta.Title: "Multisite setup in Umbraco"
+product: "CMS"
+meta.Description: "A guide to multisite setup in Umbraco"
+---
+
+# Multisite setup on Umbraco Cloud
+
+This tutorial explains how to host multiple sites from one project/installation of Umbraco.
+For practical reasons, we recommend using [Baselines](../../Umbraco-Cloud/Getting-Started/Baselines/) on the Cloud projects.
```
Comment (sofietoft) [inline Tutorials/Multisite-Setup/index.md:11]: Perhaps we could add a bit more context, as to why we recommend using the Baseline feature for Cloud projects? Could be something about the limitations to hostnames, content nodes or resources in general? 🤔

File: Tutorials/Multisite-Setup/index.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+## Structuring your website
+
+The best way to handle a multisite solution is to have multiple root nodes in the Content section, where each root node would act as a separate website.
+
+Keep in mind all the websites in your solution will be using the same schema - meaning, in most cases, your content pages on website A will be using the same properties as on website B. 
+
+## Binding the hostnames to the project
+
+Before you can map your hostnames to individual websites in the solution, you should add them in the Hostnames section on the Cloud portal to ensure they are secured with TLS.
```
Comment (sofietoft) [inline Tutorials/Multisite-Setup/index.md:22]:
```suggestion
Before you can map your hostnames to individual websites in the solution, you should add them in the Hostnames page on the Cloud portal to ensure they are secured with TLS.
```

File: Tutorials/Multisite-Setup/index.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+![Adding hostnames to the project](images/1-addinghostnames.png)
+
+Keep in mind the [hostnames have to be configured in a specific way.](../../Umbraco-Cloud/Set-Up/Manage-Hostnames/).
+
+## Mapping the hostnames to individual websites/root nodes
```
Comment (sofietoft) [inline Tutorials/Multisite-Setup/index.md:28]: How about using a list format for the steps outlined in this section?

File: Tutorials/Multisite-Setup/index.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+![SWATO site](images/7-swato.png)
+
+## Best practices
+
+While such a setup might be handy, it also comes with drawbacks.
+It is important to keep in mind that having multiple sites on one Umbraco project:
+- might increase resource usage
+- could interfere with editors' workflows, especially if there are multiple people working on both websites at once
+(that is because the solution will still use one database for both websites)
+- limit your options in regards to developing new features and making schema changes
```
Comment (sofietoft) [inline Tutorials/Multisite-Setup/index.md:53]: Looks like the linter is complaining about a sentence that's too long here 🤔 
Perhaps the bullet list needs to be structured differently... Also, make sure that each bullet starts with a capital letter.

File: Tutorials/Multisite-Setup/index.md [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+
+In general, while on the Cloud, we instead recommend using the [Baseline](../../Umbraco-Cloud/Getting-Started/Baselines/) functionality - which comes with added benefits, and offers increased stability compared to the multisite solution in a single project.
```
Comment (sofietoft) [inline Tutorials/Multisite-Setup/index.md:55]: Something odd about this sentence 🤔 
"on the Cloud, we *instead* recommend" .. Isn't this guide meant especially for Cloud?

**Review by sofietoft — COMMENTED** [review]
Super tutorial @Jay-umbr ! 😄 💪 

I've added a few suggestions here and there. Mainly structural things.
Also, let's have a chat about where we want to place this article - I've been thinking that it might make sense to have in the Cloud section after all 🤔

File: Tutorials/Multisite-Setup/index.md (hunk 6 above)
Comment (Jay-umbr) [reply Tutorials/Multisite-Setup/index.md]: fixed

File: Tutorials/Multisite-Setup/index.md (hunk 2 above)
Comment (Jay-umbr) [reply Tutorials/Multisite-Setup/index.md]: noted

File: Tutorials/Multisite-Setup/index.md (hunk 5 above)
Comment (Jay-umbr) [reply Tutorials/Multisite-Setup/index.md]: hopefully fixed now

File: Tutorials/Multisite-Setup/index.md (hunk 4 above)
Comment (Jay-umbr) [reply Tutorials/Multisite-Setup/index.md:36]: done

File: Tutorials/Multisite-Setup/index.md [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+Before you can map your hostnames to individual websites in the solution, you should add them in the Hostnames page on the Cloud portal to ensure they are secured with TLS.
+
+![Adding hostnames to the project](images/1-addinghostnames.png)
+
+Keep in mind the [hostnames have to be configured in a specific way.](../../Umbraco-Cloud/Set-Up/Manage-Hostnames/).
+:::
+
+## Mapping the hostnames to individual websites/root nodes
+
+At this point you should have several root nodes, where each is a separate website. Once the hostnames are protected by TLS:
```
Comment (sofietoft) [inline Tutorials/Multisite-Setup/index.md:38]: The "Once the hostnames are protected by TLS" is very Cloud specific, right?
Would it make sense to simply leave it out? Or perhaps say something along these lines instead:
"Once you've configured DNS and hostnames follow these steps:"

What do you think?

```suggestion
At this point you should have several root nodes, where each is a separate website. Once the hostnames are protected by TLS:
```

Comment (Jay-umbr) [reply Tutorials/Multisite-Setup/index.md]: Done!

**Pull request merged**